python/salpytools/states.py

```diff
@@ -74,11 +74,11 @@
 state_matrix[3][4] = True
 state_matrix[5][1] = True
 
-# Set up same state transitions as allowed
-state_matrix[0][0] = True
-state_matrix[1][1] = True
-state_matrix[2][2] = True
-state_matrix[3][3] = True
-state_matrix[4][4] = True
-state_matrix[5][5] = True
-state_matrix[6][6] = True
+# Disallow same state transitions
```

Comment:
Seems like "Disallowing" should involve setting to False, not commented out setting to True.

Reply:
The `state_matrix` matrix context are all set to False by default. So they need to be change to `True`